journals/journal_abbreviations_general.csv

```diff
@@ -28,1779 +26,756 @@ ACM Transactions on Knowledge Discovery from Data;ACM Trans. Knowl. Discovery Da
 ACM Transactions on Management Information Systems;ACM Trans. Manage. Inf. Syst.;;
 ACM Transactions on Mathematical Software;ACM Trans. Math. Software;;
 ACM Transactions on Multimedia Computing Communications and Applications;ACM Trans. Multimedia Comput. Commun. Appl.;;
-ACM Transactions on Parallel Computing;ACM Trans. Parallel Comput.;;
 ACM Transactions on Programming Languages and Systems;ACM Trans. Program. Lang. Syst.;;
```

Comment:
I think, your script does not read `journal_abbreviations_webofscience-dots.csv` - or keeps different abbreviations.

![image](https://user-images.githubusercontent.com/1366654/230618699-c87e75b4-ec2f-4ac9-b284-5a750d3c7d5f.png)

Could you update the script to be more aggressive?

If the journal name appears in another list, remove it from the `journal_abbreviations_general.csv`.

Reply:
Yes as you had mentioned I will make the script criteria a bit tighter where I will check only for the Title and if the Title is common i will remove the entry from the CSV file.

Reply:
Sounds good; lets check, how the result looks like.

You could output the entries, where in the general list the abbreviation is shorter than in the other ones. (but still proceed with removing - I would like to see the general list being very small - maybe, we can even delete it)

Reply:
This issue was basically due to the case disparity of the Title. I will handle that and raise a new PR.

Reply:
This issue has been handled